bot/exts/utilities/bookmark.py

```diff
@@ -153,85 +102,39 @@ def build_error_embed(message: str) -> discord.Embed:
+            log.info(
+                f"{interaction.user} tried to bookmark a message in #{interaction.channel} but doesn't have permission."
+            )
+            embed = self.build_error_embed("You don't have permission to view this channel.")
             await interaction.response.send_message(embed=embed)
             return
 
-        bookmark_title_form = BookmarkTitleSelectionForm(
-            message=message,
-            action_bookmark_function=self.action_bookmark
-        )
+        bookmark_title_form = BookmarkForm(message=message)
         await interaction.response.send_modal(bookmark_title_form)
 
     @commands.group(name="bookmark", aliases=("bm", "pin"), invoke_without_command=True)
     @commands.guild_only()
     @whitelist_override(roles=(Roles.everyone,))
-    async def bookmark(
-        self,
-        ctx: commands.Context,
-        target_message: Optional[WrappedMessageConverter],
-        *,
-        title: str = "Bookmark"
-    ) -> None:
-        """
-        Send the author a link to the specified message via DMs.
-
-        Members can either give a message as an argument, or reply to a message.
-
-        Bookmarks can subsequently be deleted by using the `bookmark delete` command in DMs.
-        """
-        target_message: Optional[discord.Message] = target_message or getattr(ctx.message.reference, "resolved", None)
-        if target_message is None:
-            raise commands.UserInputError(MESSAGE_NOT_FOUND_ERROR)
-
-        permissions = ctx.channel.permissions_for(ctx.author)
-        if not permissions.read_messages:
-            log.info(f"{ctx.author} tried to bookmark a message in #{ctx.channel} but has no permissions.")
-            embed = Bookmark.build_error_embed("You don't have permission to view this channel.")
-            await ctx.send(embed=embed)
-            return
-
-        await self.action_bookmark(ctx.channel, ctx.author, target_message, title)
-
-        view = SendBookmark(self.action_bookmark, ctx.author, ctx.channel, target_message, title)
-        embed = self.build_bookmark_embed(target_message)
-
-        await ctx.send(embed=embed, view=view)
+    @commands.cooldown(1, 30, commands.BucketType.channel)
+    async def bookmark(self, ctx: commands.Context) -> None:
+        """Teach the invoker how to use the new context-menu based command for a smooth migration."""
+        response = await ctx.guild.get_channel(Channels.sir_lancebot_playground).send(
+            f"{ctx.author.mention} The bookmark text command has been replaced with a context menu command!\n\n"
+            "To bookmark a message simply right-click (press and hold on mobile) "
+            "on a message, open the 'Apps' menu, and click 'Bookmark'."
+        )
+        await ctx.send(
+            embed=self.build_error_embed(
+                f"See [this message]({response.jump_url}) for information on how to bookmark messages."
+            )
+        )
```

Comment:
The message redirecting the user is about the same size as the information message itself, so I feel it would make more sense to just send the information directly to the channel where it's invoked- it doesn't seem too noisy.

(There's also currently a slightly weird case currently when using the command in `sir_lancebot_playground` as it sends the info message and then an embed directly below linking to the message above.

Reply:
Yea I like this suggestion, pushed the change.